Tests/Realm.Tests/Database/RemoveTests.cs

```diff
@@ -153,6 +153,38 @@ public void RemoveAllObjectsShouldClearTheDatabase()
             Assert.That(_realm.All<AllTypesObject>().Count(), Is.EqualTo(0));
         }
 
+        [Test]
+        public void RemoveAllObjectsShouldClearEntireDatabaseWithPartialSchema()
+        {
+            var c1 = new RealmConfiguration
+            {
+                Schema = new[] { typeof(Person), typeof(IntPrimaryKeyWithValueObject) }
```

Comment:
Maybe not best classes to choose but unfortunately many other classes in schema reference other classes which would make it more confusing as it'd require those referenced classes to be included in schema too